src/static/js/routes/index.js

```diff
@@ -7,193 +7,245 @@
 
 var $ = require( 'jquery' );
 
-window.requestNextAnimationFrame = require( 'requestNextAnimationFrame' );
+// Multiplier determines how pixel dense the canvas should be.
+// This way canvas can look good even on retina displays.
+var _multiplier = 2;
 
-function init() {
-  // Multiplier determines how pixel dense the canvas should be.
-  // This way canvas can look good even on retina displays.
-  var multiplier = 2;
+// This variable keeps track of time by counting the number of frames drawn.
+var _iterationCount = 0;
+
+// Variable to keep track of when all charts are finished animating.
+var _completeCount = 0;
+
+var _$chartsDom;
+var _charts;
+var _center;
+var _radius;
+var _arcStart;
 
-  // Boolean to end the animation loop so it's not just running continuously in the background.
-  var animationComplete = false;
+/**
+ * Initialize the progress bars if requestAnimationFrame is supported.
+ */
+function init() {
+  var containerSelector = '.project-progress';
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/c7b10a576887f757fac4f88c54ad1ba0914cfeec